packages/utils/src/__tests__/joiner/remote-joiner.ts

```diff
@@ -13,10 +14,37 @@ const container = {
   },
 } as MedusaContainer
 
+const fetchServiceDataCallback = async (
+  expand: RemoteExpandProperty,
+  pkField: string,
+  ids?: (unknown | unknown[])[],
+  relationship?: any
+) => {
+  const serviceConfig = expand.serviceConfig
+  const moduleRegistryName =
+    serviceConfig.serviceName[0].toLowerCase() +
```

Comment:
todo: I ve created an util in the product module pr `loweCaseFirst`, maybe you can pick it in your pr?

Reply:
will do